### PR TITLE
#3458 Include default MinIO credentials

### PR DESCRIPTION
**Pull request by GRomR1**
#3458 Include default MinIO credentials in sandbox console readout

# TL;DR
Include default MinIO credentials in sandbox console readout

## Type
- [ ] Bug Fix
- [x] Feature
- [ ] Plugin

## Are all requirements met?

- [ ] Code completed
- [ ] Smoke tested
- [ ] Unit tests added
- [ ] Code documentation added
- [ ] Any pending items have an associated Issue

## Complete description
When following the [steps](https://docs.flyte.org/en/latest/deployment/deployment/sandbox.html#start-the-sandbox) to setup Flyte on a demo sandbox, the console reports that 📂 The Minio API is hosted on localhost:30002. Use http://localhost:30080/minio/login for Minio console. However, in order to view the obect store, you need to use the default credentials for MinIO, which are:

username: minio
password: miniostorage

## Tracking Issue
https://github.com/flyteorg/flyte/issues/3458

### CODE REVIEW

**Comment by oliverhu** [comment]
+1 to this PR...I had to decode the secrets to get these credentials. Maybe put these in `Constants` in the same file than hard coding in the message?

**Review by pingsutw — APPROVED** [review]
